realm-elasticsearch/lib/realm/elasticsearch/gateway.rb

```diff
@@ -34,6 +23,29 @@ def health
         end
         HealthStatus.from_issues(issues)
       end
+
+      def method_missing(...)
```

Comment:
same question as before

Reply:
Here I actually want to pass everything.

Reply:
got it ! :)